### PR TITLE
Updates opinion scraper for Georgia Appeallate court

### LINKED ISSUE #429: Fix opinion scraper connection for Georgia Appeals Court
### 500 Server Error: Internal Server Error for url: https://www.gaappeals.us/docketdate/results_all.php?searchterm=11%2F12%2F2021&searchterm2=12%2F02%2F2021&submit=Search
The opinion scraper for **Georgia Appeals Court** is working intermitently. The scrape of the page works, but the connection to the opinions search page fails at random intervals; it even takes a couple of reload attempts to load the page with a browser.

However, this intermetice in the access to the court page is affecting found opinion records. For example, the court released opinions on **November 29th and the 30th**, but there aren't records from those dates on CourtListener between November 28th and December 1st.

`juriscraper.opinions.united_states.state.gactapp
`

Sentry Issue: [COURTLISTENER-1YZ](https://sentry.io/organizations/freelawproject/issues/2827572358/?referrer=github_integration)

```
HTTPError: 500 Server Error: Internal Server Error for url: https://www.gaappeals.us/docketdate/results_all.php?searchterm=11%2F11%2F2021&searchterm2=12%2F01%2F2021&submit=Search
(2 additional frame(s) were not displayed)
...
  File "juriscraper/AbstractSite.py", line 127, in parse
    self.html = self._download()
  File "juriscraper/opinions/united_states/state/gactapp.py", line 40, in _download
    return super()._download(request_dict=request_dict)
  File "juriscraper/AbstractSite.py", line 327, in _download
    self._post_process_response()
  File "juriscraper/AbstractSite.py", line 379, in _post_process_response
    self.request["response"].raise_for_status()
  File "requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
```

### PR DESCRIPTION
**Pull request by satsuki-chan**
Fixes scraping the published date for each opinion, instead of assigning the date when the opinion is scraped

The original connection error reported on Sentry hasn't been repeated since December 2nd, 2021 at 12:04. Also, the court's web site has been up and without any other reported issue since then. It apears that it was just a temporary network incident with the court's server.

Closes #429

### CODE REVIEW

**Comment by mlissner** [comment]
Can you take a look at this one, please @flooie ?

**Comment by mlissner** [comment]
Bumping this again for you, @flooie.

File: juriscraper/opinions/united_states/state/gactapp.py [hunk 1]
```diff
@@ -17,28 +17,14 @@ def __init__(self, *args, **kwargs):
...
-                mn_ago=self.a_while_ago.month,
-                day_ago=self.a_while_ago.day,
-                year_ago=self.a_while_ago.year,
-                mn=self.case_date.month,
-                day=self.case_date.day,
-                year=self.case_date.year,
-            )
+            f"http://www.gaappeals.us/docketdate/results_all.php?searchterm="
+            f"{self.a_while_ago.month:02d}%2F{self.a_while_ago.day:02d}%2F{self.a_while_ago.year}&searchterm2="
+            f"{self.case_date.month:02d}%2F{self.case_date.day:02d}%2F{self.case_date.year}&submit=Search"
```
Comment (flooie) [inline juriscraper/opinions/united_states/state/gactapp.py:25]: my only comment here is that this is just impossible to read but its not your fault.

File: juriscraper/opinions/united_states/state/gactapp.py [hunk 2]
```diff
@@ -17,28 +17,14 @@ def __init__(self, *args, **kwargs):
...
 
-    def _download(self, request_dict={}):
-        if self.test_mode_enabled():
-            # This is an arbitrary date that we need to set
-            # for our compar.json test to pass
-            self.case_date = convert_date_string("2017-08-14")
-        return super()._download(request_dict=request_dict)
-
     def _get_case_names(self):
         path = f"{self.base_path}/td[2]/text()"
```
Comment (flooie) [inline juriscraper/opinions/united_states/state/gactapp.py:29]: I would've liked to have seen python typing

File: juriscraper/opinions/united_states/state/gactapp.py [hunk 3]
```diff
@@ -48,9 +34,8 @@ def _get_download_urls(self):
         return list(self.html.xpath(path))
 
     def _get_case_dates(self):
-        return [self.case_date] * int(
-            self.html.xpath(f"count({self.base_path})")
-        )
+        path = f"{self.base_path}/td[3]/text()"
```
Comment (flooie) [inline juriscraper/opinions/united_states/state/gactapp.py:37]: No typing here.

**Review by flooie — COMMENTED** [review]
We are still missing typing in our functions, but otherwise a fine PR.

File: juriscraper/opinions/united_states/state/gactapp.py (hunk 3 above)
Comment (satsuki-chan) [reply juriscraper/opinions/united_states/state/gactapp.py:37]: I'll add it on future modifications.

**Pull request merged**